automacent-fwk-components/automacent-fwk-core/src/main/java/com/automacent/fwk/core/TestObject.java

```diff
@@ -126,6 +128,39 @@ public void setDriverManager(DriverManager driverManager) {
 		this.driverManager = driverManager;
 	}
 
+	// Debugger Address ----------------------------------------------------
+
+	private String debuggerAddress;
+
+	/**
+	 * set the debugger address
+	 * 
+	 * @param debuggerAddress
+	 */
+	public void setDebuggerAddress(String debuggerAddress) {
+		if (this.getDriverManager().getDriverManagerType().name().equals(DriverManagerType.CHROME.name())) {
+			if (!debuggerAddress.isEmpty()) {
+				this.debuggerAddress = debuggerAddress;
+				_logger.info(String.format("debuggerAddress set to %s", getDebuggerAddress()));
+			} else {
+				_logger.debug(String.format("debuggerAddress is not set since the value is empty"));
+			}
+		} else {
+			if (!debuggerAddress.isEmpty()) {
+				_logger.warn(String.format(
+						"debuggerAddress is not set since browser is not set to " + DriverManagerType.CHROME.name()));
+			}
+		}
+	}

```

Comment:
Change this to below
```
	public void setDebuggerAddress(String debuggerAddress) {
		if (!debuggerAddress.isEmpty()) {
                        if (this.getDriverManager().getDriverManagerType().name().equals(DriverManagerType.CHROME.name())) {
				this.debuggerAddress = debuggerAddress;
				_logger.info(String.format("debuggerAddress set to %s", getDebuggerAddress()));
			} else {
				_logger.warn(String.format("debuggerAddress cannot be set because browser is not %s", DriverManagerType.CHROME.name()));
			}
		}
	}

```